### PR TITLE
feat: backButtonNewRecipe

### PR DESCRIPTION
**Pull request by taylorichigo**

### CODE REVIEW

File: components/recipeForms/newRecipeForm.tsx [hunk 1]
```diff
@@ -36,9 +37,12 @@ const NewRecipeForm = () => {
 
   return (
     <div className="container">
-      <a href="#" className="text-1 font-bold text-zinc-400 hover:text-black">
+      <Link
+        href="/"
```
Comment (moncapitaine) [inline components/recipeForms/newRecipeForm.tsx:41]: better solution: https://nextjs.org/docs/api-reference/next/router#routerback

Comment (taylorichigo) [reply components/recipeForms/newRecipeForm.tsx]: Done

**Review by clemens1337 — APPROVED** [review]

File: components/recipeForms/newRecipeForm.tsx [hunk 2]
```diff
@@ -34,11 +35,17 @@ const NewRecipeForm = () => {
     })
   }
 
+  const router = useRouter()
+
   return (
     <div className="container">
```
Comment (moncapitaine) [inline components/recipeForms/newRecipeForm.tsx:41]: note needed inside this component

File: components/recipeForms/newRecipeForm.tsx [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import { useForm, SubmitHandler } from 'react-hook-form'
 import { useMutation } from 'urql'
+import { useRouter } from 'next/router'
 
 interface IFormInput {
```
Comment (moncapitaine) [inline components/recipeForms/newRecipeForm.tsx:5]: isn't this a recipe? Why re-inventing the wheel?

**Comment by moncapitaine** [comment]
For me there are too many interfaces and terms that are similar. Why we are talking about dishes now?

File: components/recipeForms/newRecipeForm.tsx (hunk 2 above)
Comment (taylorichigo) [reply components/recipeForms/newRecipeForm.tsx:41]: Done

**Review by moncapitaine — APPROVED** [review]

**Pull request merged**